content/livesync/models/index.textile

```diff
@@ -0,0 +1,203 @@
+---
+title: Setup
+meta_description: "Install the Ably Models SDK to get started with creating data models to keep your frontend in sync with your backend database."
+product: livesync
+languages:
+  - javascript
+---
+
+The Ably Models SDK is a standalone SDK built on "Ably's JavaScript SDK":/getting-started/setup with full TypeScript support. It enables you to easily create live, observable data models in your frontend applications to ensure they remain synchronized with your database's state in realtime.
+
+h2(#authenticate). Authenticate
+
+An "API key":/auth#api-keys is required to authenticate with Ably. API keys are used either to authenticate directly with Ably using "basic authentication":/auth/basic, or to generate tokens for untrusted clients using "token authentication":/auth/token.
+
+<aside data-type='important'>
+<p>The examples use "basic authentication":/auth/basic to demonstrate features for convenience. In your own applications, basic authentication should never be used on the client-side as it exposes your Ably API key. Instead use "token authentication":/auth/token. </p>
+</aside>
+
+"Sign up":https://ably.com/sign-up to Ably to create an API key in the "dashboard":https://ably.com/dashboard or use the "Control API":/account/control-api to create an API programmatically.
+
+API keys and tokens have a set of capabilities assigned to them that specify which operations, such as @subscribe@ or @publish@ can be performed on which resources. To use the Models SDK, the API key requires the following "capabilities":/auth/capabilities
+
+* @Subscribe@ for the channels you intend to subscribe to.
```

Comment:
`subscribe` should be lowercase as it refers to a specific capability string. Same for `history` below

Reply:
Fixed in this [commit](https://github.com/ably/docs/pull/2082/commits/c112ade0468d905e588f441cff4a895bd9b30ace)